specificatie/zoek-personen.yaml

```diff
@@ -26,6 +26,7 @@ paths:
         2.  Zoek met geslachtsnaam, voornamen en gemeente van inschrijving
             - gemeenteVanInschrijving
             - geslachtsnaam (minimaal 2 karakters, [wildcard toegestaan](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/wildcard.feature))
```

Comment:
eigenlijk klopt deze toelichting niet. er is alleen een minimum van 2 karakters bij het gebruik van een wildcard

Reply:
Je hebt helemaal gelijk, ondanks dat je niet hebt aangegeven dat dit gewijzigd moest worden heb ik dat toch gedaan. In plaats van 
> (minimaal 2 karakters, [wildcard toegestaan](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/wildcard.feature)) 

staat er nu 
>([wildcard toegestaan](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/wildcard.feature) met minimaal 2 karakters)'